lib/fonts.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const _ = require('lodash');
+const fontKeyFormat = new RegExp(/\w+(-\w*)*-font$/);
+const fontProviders = {
+    'Google': {
+        /**
+         * Parser for Google fonts
+         *
+         * @param {Array} fonts - Array of fonts that might look like
+         * Google_Open+Sans
+         * Google_Open+Sans_400
+         * Google_Open+Sans_400_sans
+         * Google_Open+Sans_400,700_sans
+         * Google_Open+Sans_400,700italic
+         * Google_Open+Sans_400,700italic_sans
+         *
+         * @returns {string}
+         */
+        parser: function(fonts) {
+            var collection = [],
+                familyHash = {};
+
+            _.each(fonts, function fontsIterator(font) {
+                var split = font.split('_'),
+                    familyKey = split[1],  // Eg: Open+Sans
+                    weights = split[2];    // Eg: 400,700italic
```

Comment:
is there another way than index? Regex would be safer + maybe a catch or something incase font doesnt have underscores

Reply:
This is existing code that I didn't spend time refactoring.

Reply:
kk